### ISSUE TITLE
Provide signed binaries and submit to snapcraft.io, Microsoft Store, Mac App Store

### ISSUE THREAD

**Issue by kriskbx** [kriskbx/gitlab-time-tracker-taskbar#22]

**Comment by bobvandevijver** [comment]
You probably need to add signing for this (I was just about to open a new ticket for that)

**Comment by kriskbx** [comment]
Yeah, I'm aware of that. Unfortunately the cheapest open source certificate for code signing still costs about 30€/year plus additional 100€/year for Apples developer program. I'm currently looking into ways how to keep this project open source but pay for my upcoming expenses. If you have experience with that, feedback or tips would be highly appreciated.

**Comment by bobvandevijver** [comment]
I have no experience with signed open source releases unfortunately 😞

**Comment by kriskbx** [comment]
Looks like I there's no other way and I gotta pay these fees to create signed binaries. I won't do that at the moment, but my plan is to setup a Patreon page in the near future for all my open source work. Hopefully I can encourage people to donate at least 10€ / month in total to cover these expenses. Maybe I implement a small popup window like Sublime Text does... I favor this approach over monetizing it in the stores because I want everyone to freely use my software. If someone enjoys it it's on his or her own to support me and further development or not.

**Comment by naturallymitchell** [comment]
You could distribute Windows packages through Chocolatey, afair, and MacOS packages through Homebrew.  I just checked and neither does code signing, but at least they're free.

**Comment by kriskbx** [comment]
I will eventually do that. But code signing is not about the distribution, but about convenience and trust:

<img width="532" alt="screen shot 2018-10-03 at 21 06 07" src="https://user-images.githubusercontent.com/3148865/46433080-3973c680-c750-11e8-9119-1f24dedb7b3f.png">

On Mac OS you have to navigate to your `/Applications` folder, right click on the app icon and select open to be able to open the application and get rid of this warning. On Windows the procedure is easier but also bugging. Besides that code signing is there to ensure that the author of a specific software distributed it and no ones else. Especially in open source everyone can build binaries and sneak some bad code in there. With code signing the users of your software know exactly that the binary was built by you.
